About

Morban15 opened the pull request:
### Issues

¿Qué se está resolviendo?
Agregando una sección about

### Todos

¿Qué queda por hacer?
Revisar y terminar

### Áreas impactadas

¿Qué páginas o componentes están afectados?


### Pasos para reproducir

¿Qué necesitamos hacer para probar?

Morban15 committed:
Add footer

Nuestro trabajo de añadir footer mejorado

Morban15 committed:
Add footer

Nuestro trabajo de añadir footer mejorado

Morban15 committed:
Agregando una rama about a nuestro nuevo proyecto

Añadimos un contenido centralizado con margin, le fijamos un alto
y un ancho.

Morban15 committed:
Agregando una rama about a nuestro nuevo proyecto

Añadimos un contenido centralizado con margin, le fijamos un alto
y un ancho.

Morban15 committed:
Merge branch 'master' of https://github.com/cosola-macile/practicar-pull-requests

Morban15 committed:
Merge branch 'master' into about

Morban15 committed:
Clean link

Cambiando /about.html por /about

Morban15 committed:
Clean link

Cambiando /about.html por /about

Morban15 committed:
Clean links

Eliminando .html del link de nuestro proyecto

Morban15 committed:
Clean links

Eliminando .html del link de nuestro proyecto

Morban15 committed:
Modificando index.html

Cambiamos la división por una sección

Morban15 committed:
Modificando index.html

Cambiamos la división por una sección

misscs commented on about/index.html:
```diff
@@ -30,14 +30,14 @@
                </ul>  
            </nav>       
-           <div class= "content">
+           <section class= "content">
```
Nice refactor!! Excelente idea cambiar el `div` para `section`

Una pequeña correction: 👀  hay un espacio  en el class atributo `class= "content`

misscs commented on about/index.html:
```diff
@@ -30,14 +30,14 @@
            </nav>       
-           <div class= "content">
+           <section class= "content">
             <h1> La caridad</h1>
```
El `h1` esta en la misma nivel que el `section` pero está dentro de `section`.  El `p` está bien aliñada 👍

misscs commented on index.html:
```diff
@@ -32,7 +32,6 @@
-          <p>Hola Mundo This is HTML5 Boilerplate.</p>
             
         </div>
         <div class = "footer">
```
Por si a caso se les olvido cometer el archivo? Este cambio no corresponden al mensaje

misscs reviewed (commented):
Parece que hemos perdido los estilos en `about/index.html` 🤔 

![captura de pantalla 2017-06-08 a la s 11 55 45](https://user-images.githubusercontent.com/51812/26938228-9b401b6a-4c41-11e7-85a6-41989bb540fa.png)

misscs commented on about/index.html:
```diff
@@ -0,0 +1,50 @@
+           <nav class="menu">  
+               <ul>  
+                    <b class="menu-li-home"><li class="menu-li"><a href="">Home</a></li></b>  
+                    <b><li class="menu-li"><a href="">About</a></li></b>  
```
No se olviden añadir el link `/about`

misscs reviewed (changes requested):
Terminé de revisar su trabajo. Sólo hay algunas pequeñas correcciones. Asegúrese de ejecutar `git pull` en la rama ` about`

Morban15 committed:
Merge branch 'about' of https://github.com/cosola-macile/practicar-pull-requests into about

Morban15 committed:
Mejorando about section

Haciendo las correcciones indicadas.

Morban15 committed:
Correcciones about section

Trabajo about section revisado y mejorado

Morban15 committed:
Quitando el espacio entre class y wrapper

Poniendo en practica las reglas aprendidas

misscs commented on about/index.html:
```diff
@@ -26,7 +26,7 @@
                <ul>  
                     <b class="menu-li-home"><li class="menu-li"><a href="">Home</a></li></b>  
-                    <b><li class="menu-li"><a href="">About</a></li></b>  
+                    <b><li class="menu-li"><a href="file:///C:/Users/Maria%20Rosa/Documents/GitHub/practicar-pull-requests/about/index.html">About</a></li></b>  
```
🤔  Creo que algo está un poquito off aquí...`file:///....` es una referencia a **tu** sistema local

misscs commented on index.html:
```diff
@@ -17,29 +17,32 @@
             
+=======
+
+>>>>>>> 7bdc8f6f4109e74250dcd923f7b65c60ff7aa362
```
No se como estas lineas lograron llegar aquí. Mañana haremos algunos experimentos para asegurarnos que git esté configurado correctamente. :) No te preocupas para hoy. Si puedes remover L34-39 👍

misscs reviewed (changes requested):
2 cambios solamente!

MariaGuillen committed:
Corrigiendo errores en about

Arreglando links y borrando lineas incorrectas

misscs committed:
Removiendo un archivo que no necesitamos

Parece que este archivo se ha agregado accidentalmente al
realizar el commit. No es gran cosa :)

misscs reviewed (dismissed):
🙌 👯 💯 Team Marias :)

misscs committed:
Merge branch 'master' into about
